app/views/moderations/_suspend_user_modal.html.erb

```diff
@@ -0,0 +1,63 @@
+<div id="suspend-unsuspend-modal" class="hidden">
+  <% if @article.user.suspended? %>
```

Comment:
I suspect that there's some refactoring we could do to reduce some duplication, but this lays a wonderful foundation.  So no changes requested just musing outloud.

Reply:
I thought of that as well... but my brain is a little fatigued 😓